rotkehlchen/db/custom_assets.py

```diff
@@ -0,0 +1,111 @@
+from typing import TYPE_CHECKING, List, Tuple
+
+from rotkehlchen.assets.asset import CustomAsset
+from rotkehlchen.assets.types import AssetType
+from rotkehlchen.db.filtering import CustomAssetsFilterQuery
+from rotkehlchen.errors.misc import InputError
+from rotkehlchen.globaldb import GlobalDBHandler
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+class DBCustomAssets:
+    def __init__(self, db_handler: 'DBHandler') -> None:
+        self.db = db_handler
+
+    @staticmethod
+    def _get_custom_assets(filter_query: CustomAssetsFilterQuery) -> List[CustomAsset]:
+        """Queries the custom_assets table using the filter query and returns a list of `CustomAsset`."""  # noqa: E501
+        query, bindings = filter_query.prepare()
+        query = 'SELECT B.identifier, B.name, A.type, A.notes from custom_assets `A` JOIN assets `B` ON A.identifier = B.identifier ' + query  # noqa: E501
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            cursor.execute(query, bindings)
+            return [CustomAsset.deserialize_from_db(entry) for entry in cursor]
```

Comment:
perhaps add a comment here that deserialize_from_db can't fail. When I first read this I was wondering why you do no error checking

Reply:
you have not done that yet. Please don't skip suggestions I make.

Reply:
The error was caught in the method that calls it below.
`get_custom_assets_and_limit_info()`

Reply:
this is not what I said. Read my comment. I said that `CustomAsset.deserialize_from_db` can't raise anything, which is why you don't raise anything here. Something like:


```suggestion
            return [CustomAsset.deserialize_from_db(entry) for entry in cursor]  # never raises
```

Also then where does `DeserializationError` ever get raised? You have it in the docstring.